packages/client/tests/functional/_utils/setupTestSuiteClient.ts

```diff
@@ -120,6 +112,7 @@ export function setupTestSuiteClientDriverAdapter({
         const queryEngineWasmFilePath = path.join(runtimeBase, 'query-engine.wasm')
         const queryEngineWasmFileBytes = await readFile(queryEngineWasmFilePath)
 
+        // @ts-expect-error (2511) - Cannot create an instance of an abstract class.
         return new globalThis.WebAssembly.Module(queryEngineWasmFileBytes)
```

Comment:
Do you have the same error here @millsp ?

Reply:
Ha, it means there are types for it finally, and it looks incorrect :smile:

Reply:
So I'll keep this here for now.
@millsp Is that something you want to take a look later? Does it need a "tech debt" issue?

Reply:
I am not sure about the issue. We could create one though with the new prioritization process I think it will never be prioritized, so maybe leave a small TODO and hope that someone will find it. I think this is an upstream problem with the type definitions provided by TS or `@types/node` and will get fixed at some point.

Reply:
Done!